DOC-425 part 6: Pindel, Pindel Annotation, SeSAMe Methylation Beta Estimation, Seurat - 10x Chromium, STAR - Counts, STAR 2-Pass Chimeric, STAR 2-Pass Genome, STAR 2-Pass Transcriptome, STAR-Fusion (#364)
NCI-GDC/gdc-docs: docs/Encyclopedia/pages/Pindel_Annotation.md

# Pindel Annotation

## Description ##

Pindel Annotation is a somatic mutation annotation pipeline used in GDC whole exome sequencing (WXS) and targeted sequencing harmonization.

## Overview ##

> Pindel Annotation is a pipeline used for WXS and targeted sequencing somatic mutation annotation at the GDC. Somatic mutation annotation is performed with Pindel Annotation using tumor and normal alignments and generates simple nucleotide variation (SNV) data.

Reviewer: ```suggestion
Pindel Annotation is a pipeline used for WXS and targeted sequencing somatic mutation annotation at the GDC. Somatic mutation annotation is performed by annotating raw VCFs, which produces annotated somatic mutation files.
```

Author: changed

### Input

> * Tumor VCF - WXS or Targeted Sequencing

Reviewer: ```suggestion
* Raw VCF - WXS or Targeted Sequencing
```

Author: changed

> * Normal VCF - WXS or Targeted Sequencing

Reviewer: ```suggestion
```

Author: deleted
